packages/client3/components/design-system/input/input-dollor.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { useForm } from "react-hook-form";
+
+import { DollarInput } from "./dollar-input";
+
+export default {
+  component: DollarInput,
+  title: "FAD/Components/DollarInput",
```

Comment:
```suggestion
  title: "FAD/Components/Input",
```

Same folder, different files